x-pack/plugins/security/server/routes/authorization/roles/delete.ts

```diff
@@ -15,6 +15,10 @@ export function defineDeleteRolesRoutes({ router }: RouteDefinitionParams) {
   router.delete(
     {
       path: '/api/security/role/{name}',
+      options: {
+        access: 'public',
```

Comment:
__question:__ do we absolutely need to add `access: 'public'` for OAS? Unfortunately, we're not ready to mark this API as public in Serverless yet (but will be in a few weeks).

Reply:
Thanks!

Reply:
I noticed our tests that validate route definition were failing, I pushed the fix to your branch, let's see if it helps.

Reply:
Thanks!